packages/usa-icon/src/test/icon.spec.js

```diff
@@ -0,0 +1,44 @@
+const assert = require("assert");
+const fs = require("fs");
+const path = require("path");
+
+const distPath = path.resolve(path.join(__dirname, "../../dist/img"));
```

Comment:
Mixed thoughts here. This can be good to test in the package, but we could also move this dist check in root.

Do you see any issues when testing the `dist` on `@uswds/uswds` releases?

Reply:
The package test checks the package dist for file/directory confirmation. If we want to have a test that checks the root dist, we can add that to the root tests at a later phase.